source/hic_hal/nxp/lpc11u35/uart.c

```diff
@@ -99,6 +99,9 @@ int32_t uart_set_configuration(UART_Configuration *config)
     baudrate = config->Baudrate;
     // Compute baud rate dividers
     mv = 15;
+#ifdef BOARD_RTL8195AM
+    config->Baudrate = 38400;
```

Comment:
Can you point me to any docs which explain why the baudrate needs to be fixed to 38400?